gpt4all-chat/src/xlsxtomd.h

```diff
@@ -0,0 +1,14 @@
+#ifndef XLSXTOMD_H
+#define XLSXTOMD_H
+
+#include <QObject>
+#include <QString>
+#include <QtGlobal>
```

Comment:
Since XLSXToMD is a standard class and not a QObject, this can just be:
```suggestion
class QString;
```